litefs/cloud-flyctl.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Configuring LiteFS Cloud with flyctl
+layout: docs
+sitemap: false
+nav: litefs
+toc: true
+---
+
+You can create and manage a LiteFS Cloud cluster via `flyctl`.
+
+## Create a LiteFS Cloud cluster
+
+```cmd
+fly litefs-cloud create
+```
+
+## Get a LiteFS Cloud token
+
+```cmd
+fly tokens create litefs-cloud --cluster $(cluster name)
+```
```

Comment:
I'm not sure how we do user data in other parts of our docs. I wonder if the `$(cluster name)` might look confusing? I'd personally go with just a `my-cluster` name to indicate that the user should replace it with their cluster name.

Reply:
As far as I've seen, we do both things in the docs (there doesn't appear to be a convention). I can switch to `my-cluster` instead!